tests/index.js

```diff
@@ -29,7 +29,7 @@ require('./testRealTimeGradingDisabled');
 require('./testShowClosedAssessmentScore');
 require('./testGradeRate');
 require('./testAccess');
-require('./testAccessAsStudent');
+// require('./testAccessAsStudent');
```

Comment:
Are all of these tests no longer needed? Or do they just need to be fixed?

Reply:
The premise of the test is no longer valid. It tests whether access is denied for a student that is not enrolled. How would you like to proceed? We can remove the test, or modify it to (1) check that enrollment was added and/or (2) check whether access is denied to a user that shouldn't have access (e.g., out of instance access date range, or invalid institution).

Reply:
> The premise of the test is no longer valid. It tests whether access is denied for a student that is not enrolled. How would you like to proceed? We can remove the test, or modify it to (1) check that enrollment was added and/or (2) check whether access is denied to a user that shouldn't have access (e.g., out of instance access date range, or invalid institution).

Can we modify it to check both (1) and (2)? 😄

Reply:
Done. (2) was done by creating a new course instance that has no access rule, than trying to acccess the assessments page for that course.